### ISSUE TITLE
instantiationDimensions misleading examples?

### ISSUE THREAD

**Issue by kieranjol** [WGBH/PBCore_2.1#36]
Hi,

I noticed that the recommendation for instantiationDimensions states

> Best practice: For physical dimensions, usage examples might be 7″ for an audio reel. When describing visual dimensions, use this for high-level descriptors such as 1080p. Use the element frameSize to describe the pixel dimensions of a visual resource.

However the examples for digital visual material all use framesize dimensions.
`<instantiationDimensions annotation="letterbox">720x480</instantiationDimensions>`
`<instantiationDimensions unitsofmeasure="pixels">100x200</instantiationDimensions>`

Perhaps examples such as 1080i, 720p, 576i would be more suitable?

**Comment by dericed** [comment]
I think I have slight preference for the `WxH` format of expression. The other examples `1080i, 720p, 576i` integrate concepts of interlacement that seem out of scope with the rest of the definition.

**Comment by kieranjol** [comment]
I agree, but I think the best practise recommendation should be changed. We would have no need for something like 1080p anyhow. I don't think we would need this element either as the info is covered in essenceFrameSize..

**Comment by iamdamosuzuki** [comment]
I agree that the inconsistency between the stated best practice and the example is problematic. However, I agree with @dericed that the `WxH` format is most relevant to the spirit of the field. The best resolution appears to be updating the wording in the best practice. 

Additionally, I think @kieranjol is right about the info truly belonging in the essence track field, but that doesn't really solve the documentation/example mismatch.

**Comment by kieranjol** [comment]
I agree with both ye.. I'm not sure what's the story with updating the guidelines, must there be a version bump?

**Comment by kieranjol** [comment]
I should clarify a vague statement I made:

> I agree, but I think the best practise recommendation should be changed. We would have no need for something like 1080p anyhow.

I just meant that we have no use in our database for recording information in the format of `1080p, 720p`. This is because we record the interlacement and more specific WidthxHeight in seperate fields.
